pyjelly/producing/producers.py

```diff
@@ -36,40 +37,43 @@ def to_stream_frame(self) -> jelly.RdfStreamFrame | None:
 
 
 @dataclass
-class FlatFrameProducer(FrameProducer):
-    _rows: list[jelly.RdfStreamRow]
-
-    def __init__(self, *, quads: bool = False) -> None:
+class ManualFrameProducer(FrameProducer):
```

Comment:
Please add some comment explaining how it works, and warning about the in-memory thing.

Reply:
Same about the remaining ones. Otherwise, I have zero idea what they are actually supposed to do, so reviewing this is not really possible.